CONTRIBUTING.md

```diff
@@ -0,0 +1,5 @@
+# Contributing
```

Comment:
Need to finish this page, once I have others parts done.

Reply:
Track this as well please